tests/test_mariadb.py

```diff
@@ -224,3 +211,30 @@ def wait_for_server():
     assert rows and len(rows) == 2
     _, num, data = rows[-1].split()
     assert num == "100" and data == "abcdef"
+
+
+def test_mariadb_healthcheck_innodb_initialized(auto_container_per_test):
+    """
+    Test if InnoDB (storage engine) has completed initializing.
+
+    See: <https://mariadb.com/kb/en/using-healthcheck-sh/#-innodb_initialized>
```

Comment:
Not a proper rst link:
```suggestion
    See: `<https://mariadb.com/kb/en/using-healthcheck-sh/#-innodb_initialized>`_
```